src/components/enketo-iframe.vue

```diff
@@ -131,7 +132,7 @@ const handleIframeMessage = (event) => {
     try { eventData = JSON.parse(event.data); } catch {}
 
     if (eventData?.enketoEvent === 'submissionsuccess') {
-      if (props.actionType === 'public-link' && redirectUrl.value) {
+      if ((props.actionType === 'public-link' || props.actionType === 'new') && redirectUrl.value) {
```

Comment:
this makes `return_url` / `returnUrl` query parameter functional for the logged in users (non public link)

Reply:
it is dependent on end of behavior (single/multiple), I can create a separate PR but that change should be released in the patch as well.

Reply:
It seems reasonable to include it in the same PR given that it all has to do with behavior post-submission. :+1: